packages/components/src/card/get-padding-by-size.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { css, type SerializedStyles } from '@emotion/react';
+
+/**
+ * Internal dependencies
+ */
+import type { Props, SizeToken } from './types';
+import { space } from '../utils/space';
+
+const CONFIG = {
+	cardPaddingXSmall: `${ space( 2 ) }`,
+	cardPaddingSmall: `${ space( 4 ) }`,
+	cardPaddingMedium: `${ space( 4 ) } ${ space( 6 ) }`,
+	cardPaddingLarge: `${ space( 6 ) } ${ space( 8 ) }`,
+};
+
+const xSmallCardPadding = css`
+	padding: ${ CONFIG.cardPaddingXSmall };
+`;
+
+export const cardPaddings = {
+	none: css`
+		padding: 0;
+	`,
+	large: css`
+		padding: ${ CONFIG.cardPaddingLarge };
+	`,
+	medium: css`
+		padding: ${ CONFIG.cardPaddingMedium };
+	`,
+	small: css`
+		padding: ${ CONFIG.cardPaddingSmall };
+	`,
+	xSmall: xSmallCardPadding,
+	// The `extraSmall` size is not officially documented, but the following styles
+	// are kept for legacy reasons to support older values of the `size` prop.
+	extraSmall: xSmallCardPadding,
+};
```

Comment:
`cardPaddings` is already essentially acting like a constant, so internally referencing `CONFIG` seems like unnecessary indirection over just inlining the values.


```suggestion
const xSmallCardPadding = css`
	padding: ${ space( 2 ) };
`;

export const cardPaddings = {
	none: css`
		padding: 0;
	`,
	large: css`
		padding: ${ space( 6 ) } ${ space( 8 ) };
	`,
	medium: css`
		padding: ${ space( 4 ) } ${ space( 6 ) };
	`,
	small: css`
		padding: ${ space( 4 ) } ${ space( 4 ) };
	`,
	xSmall: xSmallCardPadding,
	// The `extraSmall` size is not officially documented, but the following styles
	// are kept for legacy reasons to support older values of the `size` prop.
	extraSmall: xSmallCardPadding,
};
```

Reply:
Fixed with 5c873be15d550f3f51bfa64aa8a6c93746b90bb3